src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,23 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const yearOne = 15;
+  const yearTwo = 9;
+  const n = number - 2;
+  const catYear3 = n * 4;
+  const dogYear3 = n * 5;
+  let catYearTotal = yearOne + yearTwo + catYear3;
+  let dogYearTotal = yearOne + yearTwo + dogYear3;
+  if (number === 2) {
+    let catYearTotal = 24;
```

Comment:
When you reassign a variable you do not have to use the let keyword. 
let and const is for creating a variable, when you reuse it (reassign it) it is already created.
You can delete the let on line 10, 11, 15 and 16